vagrant/boot_cluster.sh

```diff
@@ -4,14 +4,15 @@ set -ex
 
 vagrant/run_toxiproxy.sh &
 
-sleep 5
+while ! nc -q 1 localhost 2181 </dev/null; do echo "Waiting"; sleep 1; done
```

Comment:
There are a couple of different tools for doing something like this, but I had to try a couple until I found one that was available on travis *and* in the default vagrant box :)